internal/dbfs_test.go

```diff
@@ -26,46 +34,82 @@ func TestAccDbfsUtilities(t *testing.T) {
 	rand.Seed(time.Now().UnixNano())
 	in := make([]byte, 1.44*1e6)
 	_, _ = rand.Read(in)
-	h := fnv.New32a()
-	h.Write(in)
-	inHash := h.Sum32()
-
-	err := w.Dbfs.Overwrite(ctx, path, bytes.NewReader(in))
-	require.NoError(t, err)
 
 	defer w.Dbfs.Delete(ctx, dbfs.Delete{
 		Path: path,
 	})
 
-	// Download directly [io.Reader] and let [io.ReadAll] determine buffer size.
+	// Upload through [io.Writer].
+	{
```

Comment:
They could be separate tests, but we would need to create the fixture in each.

I use the blocks to scope the cases. Moving around code and mixing `:=` and `=` becomes a chore otherwise.